src/profile-editor.c

```diff
@@ -849,7 +849,58 @@ terminal_profile_edit (TerminalProfile *profile,
 
 	terminal_profile_editor_focus_widget (editor, widget_name);
 
+  w = GTK_WIDGET (gtk_builder_get_object (builder, "profile-editor-notebook"));
+  gtk_widget_add_events (w, GDK_SCROLL_MASK);
+  g_signal_connect (w,
+                    "scroll-event",
+                    G_CALLBACK (terminal_profile_edit_dialog_page_scroll_event_cb),
+                    NULL);
+
```

Comment:
Use always four spaces per indent level

Reply:
Done,

I see here:
![Capture du 2020-06-28 10-55-02](https://user-images.githubusercontent.com/50292710/85943222-f481c180-b92e-11ea-8ea3-08afb0c559fc.png)
When i have this in my local repository:
![Capture du 2020-06-28 10-55-45](https://user-images.githubusercontent.com/50292710/85943228-06fbfb00-b92f-11ea-99ea-6825c14c0450.png)
Note the tab on lines before 852 in my file not the case here.